assets/custom.js

```diff
@@ -0,0 +1,6 @@
+
+// 'Collections list' on hover > 'shop' link changes color
+$('.collection-container__inner').hover(function(){
+    console.log('hello world');
+    $(this).find('.inner__link-text').toggleClass('hover');
+});
```

Comment:
this can be done with CSS:
.collection-container__inner:hover .inner__link-text {
  color: whatever;
}

Remember, if it can be done with CSS, always do it with CSS